qiskit/circuit/library/generalized_gates/rv.py

```diff
@@ -36,14 +36,17 @@ class RVGate(Gate):
 
     .. math::
 
-        \newcommand{\rotationangle}{|\vec{v}|}
-        \newcommand{\sinc}{\text{sinc}}
+        \newcommand{\rotationangle}{\frac{|\vec{v}|}{2}}
             R(\vec{v}) = e^{-i \vec{v}\cdot\vec{\sigma}} =
```

Comment:
Should the factor 1/2 also be in the exponent here? In the paper that was referenced they also have it I think
```suggestion
            R(\vec{v}) = e^{-i/2 \vec{v}\cdot\vec{\sigma}} =
```

Reply:
Done.